### DISCUSSION TITLE
How are you handling CSS?

### DISCUSSION THREAD

**Discussion by mattste** [module-federation/module-federation-examples#653] [Q&A]
I have the following MF set-up:

![mf-css](https://user-images.githubusercontent.com/2335756/105649316-a43af580-5e64-11eb-9afa-6a3c65162a4d.png)

We use Tailwind across our project (design-library, app1, app2). It'd be ideal if we could have the design-library expose a stylesheet (which would be rather large) and then app1 and app2 would purge styles they don't use. Does anyone know of a way to have this work right now? Ideas on how this _could_ be made to work?

**Comment by jacob-ebey** [comment]
This isn't something you can or want to handle within a federated system. The issue boils down to "you can't / *really* don't want to be processing assets outside of the build step". It might be better to bundle your library as es6, add it as an npm module and compile it as part of your applications. This would allow you to use postcss or such to purge unused styles within your application.
